src/realm/db.cpp

```diff
@@ -2369,6 +2373,9 @@ void DB::do_begin_write()
     // should take this situation into account by comparing with '>' instead of '!='
     info->next_served = my_ticket;
     finish_begin_write();
+    if (m_logger) {
+        m_logger->log(util::Logger::Level::trace, "writemutex acquired");
```

Comment:
wont this log statement come right after the other "writemutex released" from last statement in finish_begin_write()? So it's wasteful?

Reply:
I don't think there is such a statement in finish_begin_write() ?

Reply:
ahh - sorry. too many lines were collapsed :-)